src/sentry/monkey/__init__.py

```diff
@@ -59,6 +114,7 @@ def patch_celery_imgcat():
 
 for patch in (
     patch_httprequest_repr,
+    patch_django_runserver,
```

Comment:
Wondering if we should only monkey this during tests, or is this an actual bug that will cause us issues in Django 2.0 as well?

Reply:
No, this doesn't affect production at all. I was thinking about tossing it into conftest, but it can be removed really soon (Django 2.1) so I think it's fine the way it is now.